packages/url/package.json

```diff
@@ -19,7 +19,8 @@
 	"main": "build/index.js",
 	"module": "build-module/index.js",
 	"dependencies": {
-		"@babel/runtime": "^7.0.0-beta.52"
+		"@babel/runtime": "^7.0.0-beta.52",
```

Comment:
Aside: Why is `@babel/runtime` a dependency for this package?

Reply:
We put it as a dependency for all packages that have transpiled code published to npm. We didn't find a better solution so far. This is what Babel docs recommend:

> NOTE - Production vs. development dependencies
> 
> In most cases, you should install `@babel/plugin-transform-runtime` as a development dependency (with --save-dev).
> 
> `npm install --save-dev @babel/plugin-transform-runtime`
>
> and `@babel/runtime` as a production dependency (with --save).
>
> `npm install --save @babel/runtime`
>
> The transformation plugin is typically used only in development, but the runtime itself will be depended on by your deployed/published code. See the examples below for more details.

https://new.babeljs.io/docs/en/next/babel-plugin-transform-runtime.html#installation